content/python/concepts/assert/assert.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'assert'
+Description: 'A keyword for confirming the truthiness of a given statement.'
```

Comment:
My understanding is that the `Description` differs between term entries and concept entries (which is what this is). For concept entries, they should include (preferably begin with) the concept-by-name. For example:

```suggestion
Description: 'The assert keyword confirms the truthiness of a given statement.'
```